db/dbQuery.js

```diff
@@ -0,0 +1,28 @@
+import { db } from "./dbConnection.js";
+
+export function getAll() {
```

Comment:
It's a good idea this file! But you duplicate the request with the `routes/todos.js` file. We should use these methods in the `routes/todos.js` file too.